docs/organizations/overview.mdx

```diff
@@ -33,6 +33,8 @@ All members of an organization, regardless of their role, can view information a
 
 Administrators can [invite other users to join the organization](/docs/organizations/inviting-users). An invitation email is sent out, and organization invitations support adding existing users of your application, or new ones. New users can register once they accept the invitation.
 
+Administrators can also use [verified domains](/docs/organizations/verified-domains) to enhance how users can join organizations. Once a verified domain is added to an organization, administrators can choose between different enrollment modes: automatic invitations and automatic suggestions.
```

Comment:
Maybe don't list the enrollment modes here as it's high likely that we will forget to also update this copyright

Reply:
Like this ?

```suggestion
Administrators can also use [verified domains](/docs/organizations/verified-domains) to enhance how users can join organizations. Once a verified domain is added to an organization, administrators can choose between different enrollment modes that will determine how new members will be added.
```

Reply:
Yeah i believe this is better and less error prone for the future

Reply:
makes sense, i will update this